docs/src/guides/Parameters.jl

```diff
@@ -0,0 +1,88 @@
+# # Handling parameters
+
+# This guide shows how to overwrite the default free parameters values and
+# how to dispatch between different parameterization options based on parameters types.
+
+# ## Overwriting parameters
+
+# `CloudMicrophysics.jl` is designed to allow easy parameter calibrations.
+# As a result, free parameters are not hard-coded in the source code but are insead
+# passed as arguments to functions. The default values are stored in a separate
+# repository [CLIMAParameters.jl](https://github.com/CliMA/CLIMAParameters.jl) in a `toml` file.
+
+# We start by importing the `CLIMAParameters` package and the needed
+# `CloudMicrophysics.jl` modules. We define the precision type.
+
+import CLIMAParameters as CP
+import CloudMicrophysics.Parameters as CMP
+import CloudMicrophysics.Microphysics1M as CM1
+import CloudMicrophysics.Microphysics2M as CM2
+import CloudMicrophysics.HetIceNucleation as CMI_het
+
+FT = Float32
+
+nothing #hide
+
+# We define the default parameters struct using the default `toml` file via the constructor
+# provided in the `CloudMicrophysics.jl` `Parameters` module.
+# Additionally, we create a `toml` file in which we save the name, value and type
+# of the parameter we are changing. In a typical application, this step would be done
+# via the calibration algorithm searching for optimal parameters values.
+# We create a second struct with changed parameters based on the `override_dict.toml` file.
+# Finally we check the values of the rain formation timescales and the corresponding
+# rain formation rates.
+const default = CMP.Rain(FT)
+
+override_file = joinpath("override_dict.toml")
+open(override_file, "w") do io
```

Comment:
This could be a dict, not sure what is preferable:
```julia
override_dict = Dict(
"[rain_autoconversion_timescale]" => Dict(
        "value" => 1,
        "type" => "float",
     )
)
```